code/modules/unit_tests/icon_state_sanity.dm

```diff
@@ -0,0 +1,15 @@
+/datum/unit_test/icon_state_sanity/Run()
+	var/list/all_objs = typesof(/obj)
+	var/list/all_turfs = typesof(/turf)
+	var/list/all_mobs = typesof(/mob)
+	var/list/atoms_to_check = all_mobs | all_objs | all_turfs
+	for(var/atom_path in atoms_to_check)
+		if(isnull(atom_path))
```

Comment:
How would it be?

Reply:
This is code from earlier where this kept happening for no real reason, it shouldnt